use-cases/NIC-passthrough-and-Kata.md

```diff
@@ -0,0 +1,292 @@
+# How To Pass a Physical NIC Into a Kata Container
+![](https://github.com/Yuval-Ai/documentation/blob/doc-create-NIC-passthrough-and-Kata/use-cases/images/NIC%20passthrough%20Diagram.png)
```

Comment:
This URL points to a personal repository, could you update/change it for the one that will be here.

Reply:
and you can make it a relative link - just use a ./dir/file.jpg etc. - that should work, and is then more portable etc.

fyi, I use [grip](https://github.com/joeyespo/grip) locally to preview docs I am working on before pushing - helps my turnaround time.